spec/helpers/batch_edits_helper_spec.rb

```diff
@@ -7,21 +7,27 @@
     context "with my works" do
       it "shows the check all dropdown" do
         allow(controller).to receive(:params).and_return(controller: "my/works")
-        expect(helper.render_check_all).to have_css("span.glyphicon-cog")
+        expect(helper.render_check_all).to have_css("span.caret")
```

Comment:
can you set the subject of description block to be `subject { helper.render_check_all }` then we can do `expect(subject).to have_css...` This way, the method is only run a single time rather than 3.  Do the same with the other blocks below.